river/View.zig

```diff
@@ -494,6 +494,9 @@ pub fn map(self: *Self) !void {
         if (self.getTitle()) |s| handle.setTitle(s);
         if (self.getAppId()) |s| handle.setAppId(s);
 
+        self.pending.tags = server.config.getSpawnTagsFilter(self);
```

Comment:
```suggestion
        if (server.config.getSpawnTagsFilter(self)) |tags| self.pending.tags = tags;
```

Initial tags are set in `.init()`